src/services/git-service.ts

```diff
@@ -0,0 +1,51 @@
+import * as cp from 'node:child_process'
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+
+import { CLIError } from '@oclif/errors'
+
+import debugFactory from './debug-factory-service'
+import FSService, { ServiceParams } from './fs-service'
+
+export class GitService {
+  private static debug = debugFactory(GitService)
+  private readonly options: ServiceParams
+  private readonly fsService: FSService
+
+  constructor(options: ServiceParams) {
+    this.options = options
+    this.fsService = new FSService(options)
+  }
+
+  public createGitignore(): void {
```

Comment:
this just creates a file from a template as others so it's not worth to be here, remove it

Reply:
`createGitignore` method moved back to `InitializerService` class